src/templates/main.wxs.mustache

```diff
@@ -1,4 +1,4 @@
-<?xml version='1.0' encoding='windows-1252'?>
+<?xml version='1.0' encoding='utf-8'?>

```

Comment:
I have not looked. Can we please verify that `utf-8` is supported for Legacy and Modern WiX? MSI is a Windows specific technology, so the `windows-1252` encoding makes sense. I also have a really vague memory having issues with encodings when I initially created this crate back in 2017. It might be unrelated or something else.

Reply:
I tested `utf-8` is supported for both, in fact Modern WiX removes that header altogether now. For most windows stuff we've started to standardize on utf-8 (before it was utf-16) except in windows.h/WDK stuff, but that's a whole other can of worms. I was actually surprised windows-1252 wasn't supported by default because that's been there since the beginning, but considering it's XML and that's meant for the web I guess it sort of makes sense to migrate away from it. I'll do a thorough check though, is it enough to check from windows 11? Or would you like me to verify an older OS version?

Reply:
Awesome! Thank you for verifying.